sdk/keyvault/azure-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/Action.java

```diff
@@ -32,7 +32,7 @@ LifetimeActionType actionType() {
      * @param lifetimeActionType the lifetimeActionType value to set
      * @return the Action object itself.
```

Comment:
the **updated** Action object.

Reply:
can lifetimeActionType be null?

Reply:
Internal class, doesn't matter.
But fixed it.
It can be null.